core/domain/summary_services_test.py

```diff
@@ -196,6 +196,103 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
         self.assertDictContainsSubset(expected_summary,
                                       displayable_summaries[0])
 
+    def test_get_displayable_exp_summary_dicts(self):
+        # A list of exp_id and None objects are passed in:
+        # Should only return exploration with exp_id
+
+        exp_summaries_partial_list = (
+            [None] + exp_services.get_exploration_summaries_matching_ids(
+                [self.EXP_ID_2]))
+        exploration_summary_dicts = (
+            summary_services.get_displayable_exp_summary_dicts(
+                exp_summaries_partial_list))
+        expected_summary_dict = {
+            'category': u'A category',
+            'community_owned': False,
+            'human_readable_contributors_summary': {
+                self.ALBERT_NAME: {
+                    'num_commits': 2,
+                    'profile_picture_data_url': None
+                }
+            },
+            'id': self.EXP_ID_2,
+            'language_code': feconf.DEFAULT_LANGUAGE_CODE,
+            'num_views': 0,
+            'objective': u'An objective',
+            'ratings': feconf.get_empty_ratings(),
+            'status': u'public',
+            'tags': [],
+            'title': u'Exploration 2 Albert title',
+            'thumbnail_bg_color': '#05a69a',
+            'thumbnail_icon_url': '/images/gallery/thumbnails/Lightbulb.svg'
+        }
+        self.assertEqual(len(exploration_summary_dicts), 1)
+        self.assertDictContainsSubset(expected_summary_dict,
+                                      exploration_summary_dicts[0])
+
+
+class ExplorationCategories(
```

Comment:
GalleryCategoriesTest

Though, I've just noticed, there's something a bit weird about this file compared to the others in the Oppia codebase -- the class names don't end in "Test", although they should. Could you please add that suffix to the other class names in this file as well?